enclave-manager/web/packages/app/src/emui/enclaves/components/widgets/ImageButton.tsx

```diff
@@ -13,25 +28,176 @@ function getUrlForImage(image: string): string | null {
     return `https://hub.docker.com/r/${imageParts[0]}/${imageParts[1]}`;
   }
   // Currently no other registries supported
-  return null;
+  return;
 }
 
+function objectToStarlark(o: any, indent: number) {
```

Comment:
this function is adapted from this function here: https://github.com/kurtosis-tech/kurtosis/blob/86acc7f757d23ff65bc20af4b8a250846cee693e/enclave-manager/web/packages/app/src/emui/enclaves/components/enclaveBuilder/utils.ts#L307